### PR TITLE
Feature/san 5258 balance view

### PR DESCRIPTION
**Pull request by GarageInc**
Need to make a review after merge of https://github.com/santiment/app/pull/361 (related changes for trigger button)
image
image

### CODE REVIEW

File: src/ducks/HistoricalBalance/page/HistoricalBalancePage.module.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@import "~@santiment-network/ui/mixins.scss";
+@import 'variables.scss';
```
Comment (DmitriVanGuard) [inline src/ducks/HistoricalBalance/page/HistoricalBalancePage.module.scss]: Where is located this `'variables.scss'`?

File: src/ducks/HistoricalBalance/balanceView/BalanceView.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import { Input } from '@santiment-network/ui'
+import cx from 'classnames'
+import GetHistoricalBalance from '../GetHistoricalBalance'
+import HistoricalBalanceChart from '../chart/HistoricalBalanceChart'
+import AssetsField from '../AssetsField'
+import styles from './BalanceView.module.scss'
+import {
```
Comment (haritonasty) [inline src/ducks/HistoricalBalance/balanceView/BalanceView.js]: styles to bottom

File: src/ducks/HistoricalBalance/balanceView/BalanceView.module.scss [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+.signalTrigger {
+  color: var(--jungle-green);
+  background-color: var(--athens);
+  border: 1px solid var(--jungle-green);
+  fill: var(--jungle-green);
+}
+
+@include responsive('phone', 'phone-xs') {
+  .header {
+    flex-direction: column;
```
Comment (haritonasty) [inline src/ducks/HistoricalBalance/balanceView/BalanceView.module.scss]: @include should be inside each class

File: src/ducks/HistoricalBalance/balanceView/BalanceView.module.scss [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+.InputWithAction button {
+  border-left: 1px solid #e0e4ee;
+  border-radius: 0;
+}
+
+.signalTrigger {
+  color: var(--jungle-green);
+  background-color: var(--athens);
+  border: 1px solid var(--jungle-green);
+  fill: var(--jungle-green);
```
Comment (haritonasty) [inline src/ducks/HistoricalBalance/balanceView/BalanceView.module.scss]: background should be white. Pls, remove those custom styles and change Button modificators (delete variant='flat', add 'border' and some more additional (if it needed))

File: src/ducks/HistoricalBalance/chart/HistoricalBalanceChart.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  XAxis,
+  YAxis,
+  Tooltip,
+  CartesianGrid,
+  ResponsiveContainer
+} from 'recharts'
+import { mergeTimeseriesByKey } from '../../../utils/utils'
+import { getDateFormats } from '../../../utils/dates'
+import styles from './HistoricalBalanceChart.module.scss'
+import { labelFormatter } from '../../../utils/formatting'
```
Comment (haritonasty) [inline src/ducks/HistoricalBalance/chart/HistoricalBalanceChart.js]: styles to bottom

File: src/ducks/HistoricalBalance/page/HistoricalBalancePage.module.scss [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+@include responsive('desktop') {
+  .historicalBalancePage {
+    margin-bottom: 2em !important;
+    width: $breakpoint-desktop;
+  }
+}
+
+@include responsive('phone', 'phone-xs') {
+  .header {
+    flex-direction: column;
```
Comment (haritonasty) [inline src/ducks/HistoricalBalance/page/HistoricalBalancePage.module.scss]: @include in classes

File: src/ducks/HistoricalBalance/balanceView/BalanceView.js [hunk 7]
```diff
@@ -4,13 +4,14 @@ import cx from 'classnames'
...
 import AssetsField from '../AssetsField'
-import styles from './BalanceView.module.scss'
 import {
   ETH_WALLET_AMOUNT_UP,
   ETH_WALLET_METRIC
 } from '../../Signals/utils/constants'
 import SignalMasterModalForm from '../../Signals/signalModal/SignalMasterModalForm'
 import ShowIf from '../../../components/ShowIf'
+import styles from './BalanceView.module.scss'
+import Button from '@santiment-network/ui/Button'
```
Comment (haritonasty) [inline src/ducks/HistoricalBalance/balanceView/BalanceView.js]: why Buttom below styles)

File: src/ducks/HistoricalBalance/balanceView/BalanceView.module.scss (hunk 4 above)
Comment (GarageInc) [reply src/ducks/HistoricalBalance/balanceView/BalanceView.module.scss]: Removed, changed to button types

File: src/ducks/HistoricalBalance/balanceView/BalanceView.js (hunk 7 above)
Comment (GarageInc) [reply src/ducks/HistoricalBalance/balanceView/BalanceView.js]: fixed

**Pull request merged**